applications/precompiled_pva/README.md

```diff
@@ -0,0 +1,87 @@
+# PVA-Accelerated Image Sharpening Application
+
+This application demonstrates the usage of [Programmable Vision Accelerator (PVA)](#about-pva) within a Holoscan
+application. It reads a video stream, applies an unsharp mask filter and renders it via the
+visualizer. The unsharp mask filtering operation is done in PVA. Since the PVA is used for this
+operation, the GPU workload is minimized. This example is a demonstration of how pre-processing, post-processing, and image processing tasks can be offloaded from a GPU, allowing it to concentrate on more compute-intensive machine learning and artificial intelligence tasks.
+
+This example application processes a video stream, displaying two visualizer windows: one for the original stream and another for the stream enhanced with image sharpening via PVA.
+
+## About PVA
+
+PVA is a power-efficient VLIW processor, specialized for image processing and computer vision
+algorithms. The CUPVA SDK provides a unified programming model for PVA. Contact NVIDIA to get access
```

Comment:
@tbirdso 
CUPVA is not a library for CUDA. It is an SDK which is parallel to CUDA. The name is a standalone acronym for "Compute Unified Programmable Vision Accelerator". In CUDA libraries such as cuDNN, cuBLAS etc, 'cu' is not part of the acronym but instead a designation that the library contains CUDA code. The CUPVA SDK contains no CUDA code. To make it clear that CUPVA is a full acronym and an SDK which is parallel to CUDA, we use all caps for the name.

Reply:
Thanks for the background @mdemircin, I wasn't familiar. I suggest using the full name for the first README reference in case others have the same confusion: 
```suggestion
algorithms. The Compute Unified Programmable Vision Accelerator (CUPVA) SDK provides a unified programming model for PVA. Contact NVIDIA to get access
```

Reply:
Updated. Thank you for the suggestion.